.github/workflows/windows-workflow.yml

```diff
@@ -23,6 +23,12 @@ on:
       - debian-packager/
       - bin/PCSX2_keys.ini.default
       - "pcsx2/PAD/Linux/**"
+  workflow_dispatch:
+    inputs:
+      retainDebugArtifacts:
+        description: 'Retain debug artifacts (.pdb/.exp/.lib) for an easier debugging experience'
```

Comment:
`'Retain debug artifacts (.pdb/.exp/.lib) for an easier debugging experience? (true or false)'
`
Perhaps?